raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientSnapshotTest.java

```diff
@@ -52,13 +53,17 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Stream;
 
+import static org.apache.kafka.raft.RaftClientTestContext.Builder;
+import static org.apache.kafka.raft.RaftClientTestContext.MockListener;
+import static org.apache.kafka.raft.RaftClientTestContext.RaftProtocol;
```

Comment:
Let's revert this change. It cause a lot of unnecessary line changes to this file.

Reply:
Ok.
Just for memo: `RaftProtocol` will be extracted as a single file. I will revert `Builder` and `MockListener`